task6/js/module/actionButtons.js

```diff
@@ -0,0 +1,56 @@
+const actionButtons = () => {
+	function sortByList(array) {
+		const arr = [...array];
+		arr.sort((a, b) => {
+			return b.classList.contains('selected') - a.classList.contains('selected');
+		});
+
+		ul.innerHTML = '';
+
+		for (let item of arr) {
+			ul.insertAdjacentElement('beforeend', item);
+		}
+	}
+
+	const randomToDo = () => {
+		const randomList = ['Buy a new book', 'Buy products', 'Read books', 'Learn react', 'Watch movie'];
+		return randomList[Math.floor(Math.random() * 5)];
+	};
+
+	function deleteSelectedItem(collection) {
+		[...collection].forEach((element) => {
+			if (element instanceof HTMLElement && element.classList.contains('selected')) {
+				element.remove();
+			}
+		});
+	}
```

Comment:
this code looks better
```suggestion
	function deleteSelectedItem(collection) {
		if (collection instanceof HTMLCollection && collection.length) {
			Array.from(collection).forEach(element => {
				if (element.classList.contains('selected')) {
					element.remove();
				}
			});
		}
	}
```